src/display/canvas.js

```diff
@@ -705,8 +705,9 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
   CanvasGraphics.prototype = {
 
-    beginDrawing: function CanvasGraphics_beginDrawing(transform, viewport,
-                                                       transparency) {
+    beginDrawing: function CanvasGraphics_beginDrawing({
+      transform, viewport, transparency, backgroundColor = null,
+    }) {
```

Comment:
We slowly (since not all the code covered by tests) refactoring that to new style. In sake of this PR, we prefer to change only the affected code.